### PR TITLE
contributing storage demo

### PR DESCRIPTION
**Pull request by jmspring**

### CODE REVIEW

File: incubator/storage_demo/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (SorraTheOrc) [inline incubator/storage_demo/LICENSE]: acs-demos is under MIT, any objections to relicensing as MIT?

File: incubator/storage_demo/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# acsbc_storage
+Storage demos on ACS
```
Comment (SorraTheOrc) [inline incubator/storage_demo/README.md]: Can we provide a little more description, some bullet points on what the demo achieves>

**Comment by colemickens** [comment]
@jmspring any chance you can test the PVC with the storage account name specified and maybe at least mention it or incorporate it into the demo? I don't blame you for saying no, just thought I'd ask given the thread on ACS-Talk right now.

**Comment by SorraTheOrc** [comment]
@jmspring I'd love to merge this but I need you to remove the Apache license as we are MIT licensed. I'd love a little more info in the readme about what the demo is but I can always do that myself. Licensing is something I want to be sure you approve of.

**Comment by squillace** [comment]
@jmspring repinging after a while. We want your stuff, but see ross's and Cole's comments. Don't want the stuff go wasting!!!

**Comment by jmspring** [comment]
@rgardler License was fixed a bit ago, small readme added

**Pull request merged**